src/app/views/experiment_page.js

```diff
@@ -0,0 +1,68 @@
+import View from 'ampersand-view';
+import dom from 'ampersand-dom';
+import mustache from 'mustache';
+import webChannel from '../lib/web_channel';
+
+export default View.extend({
+  template: mustache.render(`<section class="page" data-hook="experiment-page">
```

Comment:
Also looks like you are using backticks here. If I understand correctly, this looks like the triple-double of templating.